### PR TITLE
Rebalanced stats for the security family of space suits

### PR DESCRIPTION
**Pull request by TheCrimsonJupiter**
## About the PR
This PR exists to rebalance the security gear and make it more viable against station threats. Most importantly, the Warden's suit is tankier and the officer and HoS suits are now faster and have heat resistance. 

## Why / Balance
Security space gear is universally regarded as completely terrible and inadequate to handle any of the threats that the station faces on a per round basis. Heat resistance is important for any battle with space dragons, against lasers, eswords, and any of the other heat and fire based threats that security is likely to face. Radiation resistance is because random space suits like the paramedic's voidsuit have 15% resistance, and the HoS' has 50%, but Warden and the officer hardsuits are lacking any, and it wouldn't feel very consistent for an ultralight paramedic suit to have rad resistance but not the station's dedicated security team. My anecdotal evidence for this working and not causing majorly balance issues is DeltaV, where many of these stat changes were long ago implemented and it works.
## Technical details

Warden's spacesuit now fits the role of a crew version of the juggernaut. It has 65% resistance to most things, now including heat damage, and has been slowed down further to a 35% movement penalty. This suit is also pending a rename and resprite to better reflect this aesthetically, something about tanks or defense or whatever.

Head of Security's space suit has been given a 40% resistance heat, the explosion resistance has been raised to 60% to be in line with the regular officer hardsuit, and the movement has been increased to only a 15% penalty. It has been renamed to "Head of Security's Assault Suit" to reflect this speed increase, and eventual resprite.

Security hardsuits have been given 30% resistance to heat. up from the previous 0%, along with being twice as fast with only a 15% movement penalty.

**Changelog**

:cl:
- tweak: Rebalanced the security family of space suits.

### CODE REVIEW

**Comment by Zap527** [comment]
Seems like you got some commits that dont belong on your branch. Did you branch off from your previous branch instead of master by mistake?

**Comment by FluffMe** [comment]
It's fine. I'll squash anyway.

**Comment by TheCrimsonJupiter** [comment]
> Seems like you got some commits that dont belong on your branch. Did you branch off from your previous branch instead of master by mistake?

I reckon I did. My bad.

File: Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml [hunk 1]
```diff
@@ -260,17 +262,18 @@
...
-      coefficients:
-        Blunt: 0.5
-        Slash: 0.6
-        Piercing: 0.6
+        Blunt: 0.35
+        Slash: 0.35
+        Piercing: 0.35
+        Radiation: 0.7
+        Heat: 0.35
         Caustic: 0.7
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml:273]: Missing `coefficients:` causes fails.
Seems you removed an extra line by mistake

**Review by FluffMe — CHANGES_REQUESTED** [review]
- PR needs a new name.
- One issue (so far) in YAML structure.

**Review by FluffMe — CHANGES_REQUESTED** [review]
OK, errors should be dealt with now.

- As this is the Upstream file, please add a comment about the changes you've made before every entity. The comment should start with `# Harmony`
- Pretty sure this requires changelog entry, as it's very player facing.

Otherwise LGTM! Please let me know if admin team is on board with this and I'll merge.

**Review by FluffMe — DISMISSED** [review]

**Review by FluffMe — APPROVED** [review]

**Pull request merged**